py/server/deephaven/table_listener.py

```diff
@@ -72,208 +257,164 @@ def _do_locked(f: Callable, lock_type="shared") -> None:
     Raises:
         ValueError
     """
-    throwing_runnable = jpy.get_type(
-        "io.deephaven.integrations.python.PythonThrowingRunnable"
-    )
-    update_graph_processor = jpy.get_type(
-        "io.deephaven.engine.updategraph.UpdateGraphProcessor"
-    )
+    throwing_runnable = jpy.get_type("io.deephaven.integrations.python.PythonThrowingRunnable")
+    update_graph_processor = jpy.get_type("io.deephaven.engine.updategraph.UpdateGraphProcessor")
```

Comment:
Do you want these here or with the other java types?
Do you want these names or the _J names?

I don't have a strong view.  Just a question of consistency.